### DISCUSSION TITLE
Add option to hide specific beatmap's update button

### DISCUSSION THREAD

**Discussion by JamieWright99** [ppy/osu#22117] [Ideas]
I frequently edit beatmaps, for example adding videos or changing some patterns. When I do so a flashing update button appears on the song. It's hard to accidently update a map so its not a big priority however it makes the songs tab look ugly and distracting. Maybe right clicking the update icon could bring up an option to hide it.
![Song Name](https://user-images.githubusercontent.com/102170503/211631487-b09b331c-c251-434b-974f-a6899a643462.png)

**Comment by peppy** [comment]
Does this still happen if you create a new difficulty with your modifications? I'd see that as being the correct path forward, rather than editing the existing difficulties.

**Reply by JamieWright99** [reply]
I can't test adding videos at the moment but it doesn't appear when you edit map objects. I think I will use that method considering this was only a super small thing. 

Funnily enough I found a bug where creating a new difficulty and deleting them marks maps as 'Local'. Just thought to share here

https://user-images.githubusercontent.com/102170503/211767617-9f07ffb5-d271-42ba-a020-12fbc45b4170.mp4
